solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -919,7 +934,12 @@ def bar_subdivisions(df, category, metric):
     elif category == 'year':
         x_axis_kwargs = {'dtick': 1}
     elif category == 'date':
-        x_axis_kwargs = {'dtick': '1d'}
+        # Sets a '{month} {day}' tick label format when zoomed in. Plotly's
+        # default behavior is to display full date and time.
+        x_axis_kwargs = {'tickformatstops': [
+            dict(dtickrange=[None, 604800000], value='%b %e'),
```

Comment:
what does 604800000 correspond to?

Reply:
Turns out they are ms, I had no idea and got here through trial and error but after some more digging I found a plotly forum post suggesting they were milliseconds, so I've updated the comment here to avoid other people having to go down that rabbit hole.